lib/models/gigya_models.dart

```diff
@@ -32,7 +32,7 @@ class GigyaResponse {
   }
 
   GigyaResponse.fromJson(dynamic json)
-      : callId = json['callId'],
+      : callId = json['callId']?.toString(),
```

Comment:
callId in Gigya response is always string and not optional.

Reply:
I had issue with it, so I don't recommend change it